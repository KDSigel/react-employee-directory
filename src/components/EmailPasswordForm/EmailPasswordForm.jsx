export default function EmailPasswordForm() {
    return (
        <div>
            Email<input />
            Password<input />
            <button>this changes</button>
        </div>
    )
}
