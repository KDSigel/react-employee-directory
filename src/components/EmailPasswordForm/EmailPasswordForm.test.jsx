import { render } from '@testing-library/react';
import EmailPasswordForm from './EmailPasswordForm';

it('should render the Home component', () => {
    const { container } = render(<EmailPasswordForm />
      );
      expect(container).toMatchSnapshot();
    });