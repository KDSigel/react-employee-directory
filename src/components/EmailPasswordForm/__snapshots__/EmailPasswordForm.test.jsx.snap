// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Home component 1`] = `
<div>
  <form>
    <label>
      Email
      <input
        type="email"
        value=""
      />
    </label>
    <label>
      Password
      <input
        type="password"
        value=""
      />
    </label>
    <button
      type="submit"
    />
  </form>
</div>
`;
